app/assets/stylesheets/spotlight/_spotlight.scss

```diff
@@ -49,4 +49,30 @@ body {
 
 .btn .blacklight-icons svg {
   fill: var(--bs-btn-color);
+}
+
+
+.link-unstyled {
```

Comment:
Since this is only used in the header, can we move it to https://github.com/projectblacklight/spotlight/blob/0325032ece9c92cf92f5409b6558cf360397e256/app/assets/stylesheets/spotlight/_header.scss ?

Reply:
done